__tests__/pet.test.js

```diff
@@ -0,0 +1,149 @@
+const Pet = require('../src/pet');
+
+let pet;
+beforeEach(() => {
+	pet = new Pet('Fido');
+});
+describe('constructor', () => {
+	it('returns an object', () => {
+		expect(new Pet('Fido')).toBeInstanceOf(Object);
```

Comment:
Since you are already creating an instance of `Pet` in the beforeEach you dont need to do it again here, you can just reference what you've already made:
```suggestion
		expect(pet).toBeInstanceOf(Object);
```